runtime/src/tasks/qthreads/Makefile

```diff
@@ -33,6 +33,14 @@ include Makefile.share
 
 TARGETS = $(TASKS_OBJS)
 
+# define a macro containing the qthreads topology module
+
+ifeq (, $(CHPL_QTHREAD_TOPOLOGY))
+RUNTIME_DEFS += -DCHPL_QTHREAD_TOPOLOGY=hwloc	# default
+else
+RUNTIME_DEFS += -DCHPL_QTHREAD_TOPOLOGY=$(CHPL_QTHREAD_TOPOLOGY)
+endif
+
```

Comment:
We have some existing logic to store the result of qthread configure time choices in a header the runtime shim  includes:

https://github.com/chapel-lang/chapel/blob/605f81016d57a6ef9e33a1372c3ef824160e91c0/third-party/qthread/Makefile#L228-L237

I  think it'd probably be simplest (and less likely to get out of sync) to just define `CHPL_QTHREAD_TOPOLOGY_BINDERS 1` or something there and then check that in the shim.

(if we imagined making the topology choice user facing, I think we'd add a chplenv variable like `CHPL_QTHREAD_TOPOLOGY` and make it part of the unique runtime/qthread paths. I don't think we want to do that since I think we just want to use `binders` everywhere, but just sharing in case it's helpful context.)